hera_pspec/pspecdata.py

```diff
@@ -2902,7 +2905,7 @@ def pspec_run(dsets, filename, dsets_std=None, groupname=None,
             bls2_list.append(_bls2)
 
     # Open PSpecContainer to store all output in
-    psc = container.PSpecContainer(filename, mode='rw')
+    psc = container.PSpecContainer(filename, mode='rw', keep_open=False)
```

Comment:
Is it best to hard-code `keep_open=False` here? (I'm genuinely asking, and I don't know what the larger context here is.)

Reply:
yes i think its right to hardcode it as false